exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,45 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+// function isNegative(num){
```

Comment:
I see you're playing around with arrow functions here - this looks great! Well done!